tests/e2e/test_api.py

```diff
@@ -55,14 +57,24 @@
 OLS_USER_DATA_COLLECTION_INTERVAL = 10
 OLS_COLLECTOR_DISABLING_FILE = OLS_USER_DATA_PATH + "/disable_collector"
 
+# Configure logging
+logging.basicConfig(
+    level=logging.INFO, format="%(asctime)s - %(levelname)s - %(message)s"
+)
+logger = logging.getLogger(__name__)
+
 
+# ruff: noqa: S605, S607
```

Comment:
it is usually better to put these comments after lines with "problematic" commands